OpenMEEG/libs/OpenMEEGMaths/include/BlasLapackImplementations/OpenMEEGMathsMKLConfig.h

```diff
@@ -12,7 +12,11 @@
 
 
 extern "C" {
+    #ifdef MKL_ILP64
+    double dlange(const char&,const long long&,const long long&,double*,const long long&,double*);
+    #else
     double dlange(const char&,const int&,const int&,double*,const int&,double*);
+    #endif
 }
 
 #define DLANGE dlange
```

Comment:
Would it be interesting to create the proper typedef and use it in all definitions ?